libraries/aws_ec2_launch_templates.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AWSEc2LaunchTemplates < AwsResourceBase
+  name 'aws_ec2_launch_templates'
+  desc 'Verifies settings for a collection of AWS EC2 Instances'
+  example '
+    describe aws_ec2_launch_templates do
+      it { should exist }
+    end
+  '
+
+  attr_reader :table
+
+  FilterTable.create
+             .register_column(:launch_template_ids,           field: :launch_template_id)
+             .register_column(:launch_template_names,         field: :launch_template_name)
+             .register_column(:create_time,                   field: :create_time)
+             .register_column(:created_by,                    field: :created_by)
+             .register_column(:default_version_number,        field: :default_version_number)
+             .register_column(:tags,                          field: :tags)
+             .register_column(:latest_version_number,         field: :launch_template_tags)
+             .install_filter_methods_on_resource(self, :table)
+
+  def initialize(opts = {})
+    super(opts)
+    validate_parameters
+    @table = fetch_data
+  end
+
+  def fetch_data
+    launch_template_rows = []
+    pagination_options = {}
+    loop do
+      catch_aws_errors do
+        # require 'byebug'
+        # byebug
+        @api_response = @aws.compute_client.describe_launch_templates(pagination_options)
+      end
+      return [] if !@api_response || @api_response.empty?
```

Comment:
this conditional should return `launch_template_rows`, otherwise `fetch_data` would return an empty array on the second or any other consecutive loop